README.md

```diff
@@ -79,6 +79,18 @@ You must install [Msys2](https://www.msys2.org/) first, then **remember choosing
 
 The **default MSYS environment** is based on Cygwin, which **won't work**!
 
+*steps in order to build for windows:
+-Download msys2 then open the shell and type those commands respectively:
+ update pacman -Syu then pacman -su.
```

Comment:
Using backticks here will help the reader understand what exactly is to be run. See the difference:
Did you mean to run `update pacman -Syu` or `pacman -Syu`?

This was formatted by writing

```
Did you mean to run `update pacman -Syu` or `pacman -Syu`?
```